tests/observability/metrics/utils.py

```diff
@@ -1342,3 +1344,43 @@ def get_vmi_guest_os_kernel_release_info_metric_from_vm(
         NODE_STR: vm.vmi.virt_launcher_pod.node.name,
         "vmi_pod": vm.vmi.virt_launcher_pod.name,
     }
+
+
+def get_file_system_metric_mountpoints_existence(
```

Comment:
should be named something like `wait_for_file_system_metric_mountpoints_existence` to indicate that the function waits for something (`get` is not suitable as nothing is returned)